src/containers/articles/ShareArticle.js

```diff
@@ -0,0 +1,158 @@
+import React, { Component } from "react";
+import { connect } from "react-redux";
+import { apiCalls } from "../../Helpers/axios";
+import Input from "../../components/Input/Input";
+import Modal from "../../components/Modal/Modal";
+import { ShowMessage } from "../../redux/actions/SnackBarAction";
+import "../../assets/css/sharing.css";
+
+export class ShareBar extends Component {
+  state = {
+    openmodal: false,
+    email: ""
+  };
+  handleSuccessShareResponse = ({ data }) => {
+    const newLink = data.shared_link.replace(
+      "http://ah-premier-staging.herokuapp.com/api",
+      "https://ah-fortem-staging.herokuapp.com"
+    );
+    const newWindow = window.open(newLink, "_blank");
+    newWindow.focus();
+  };
+
+  handleEmailChange = event => {
+    this.setState({
+      email: event.target.value
+    });
+  };
+
+  handleMessages = (
```

Comment:
I like the way you are using a single function to handle errors

Reply:
Thank you. Reusability to avoid duplication is our main aim here.